### ISSUE TITLE
Umbraco 9.4.3 Smidge CSS notation clash

### ISSUE THREAD

**Issue by matt-tolliday-iss** [outfielddigital/konstrukt#14]
**Describe the bug**
When navigating to any list view with this version of Umbraco and the 1.0.2 of konstrukt then a clash in the Smidge minification/bundling occurs on the rgb CSS definition resulting in the same error as described here: https://github.com/madskristensen/BundlerMinifier/issues/552

**Steps To Reproduce**
1. Go to any list view
2. load the grid
4. See error in network tab (Chrome) where umbraco backoffice combined CSS from plugins fails to load

![image](https://user-images.githubusercontent.com/80101704/165805057-e7cc9af6-35fc-40c6-8cd6-873ab917f1d4.png)

Stack Trace:

![image](https://user-images.githubusercontent.com/80101704/165805825-87333258-91c4-41c1-b72b-32f7bcf225d6.png)

```
Smidge.Nuglify.NuglifyCss.ProcessAsync(FileProcessContext fileProcessContext, PreProcessorDelegate next)
Smidge.FileProcessors.PreProcessPipeline.ProcessNext(Queue<IPreProcessor> queue, FileProcessContext fileProcessContext)
Smidge.FileProcessors.PreProcessPipeline+<>c__DisplayClass5_0+<<ProcessNext>b__0>d.MoveNext()
Smidge.FileProcessors.PreProcessPipeline.ProcessNext(Queue<IPreProcessor> queue, FileProcessContext fileProcessContext)
Smidge.FileProcessors.PreProcessPipeline+<>c__DisplayClass5_0+<<ProcessNext>b__0>d.MoveNext()
Smidge.FileProcessors.CssImportProcessor.ProcessAsync(FileProcessContext fileProcessContext, PreProcessorDelegate next)
Smidge.FileProcessors.PreProcessPipeline.ProcessNext(Queue<IPreProcessor> queue, FileProcessContext fileProcessContext)
Smidge.FileProcessors.PreProcessPipeline.ProcessAsync(FileProcessContext fileProcessContext)
Smidge.FileProcessors.PreProcessManager.ProcessFileImpl(IWebFile file, BundleOptions bundleOptions, BundleContext bundleContext)
Smidge.FileProcessors.PreProcessManager.ProcessFile(IWebFile file, BundleOptions bundleOptions, BundleContext bundleContext)
Smidge.FileProcessors.PreProcessManager.ProcessAndCacheFileAsync(IWebFile file, BundleOptions bundleOptions, BundleContext bundleContext)
Smidge.Controllers.SmidgeController.Bundle(BundleRequestModel bundleModel)
Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor+TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, object controller, object[] arguments)
System.Threading.Tasks.ValueTask<TResult>.get_Result()
System.Runtime.CompilerServices.ValueTaskAwaiter<TResult>.GetResult()
Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, object state, bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(ref State next, ref Scope scope, ref object state, ref bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, object state, bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, object state, bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(ref State next, ref Scope scope, ref object state, ref bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, object state, bool isCompleted)
Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
Umbraco.StorageProviders.AzureBlob.AzureBlobFileSystemMiddleware.HandleRequestAsync(HttpContext context, RequestDelegate next)
Microsoft.AspNetCore.Builder.UseMiddlewareExtensions+<>c__DisplayClass6_1+<<UseMiddlewareInterface>b__1>d.MoveNext()
Umbraco.Cms.Web.Common.Middleware.BasicAuthenticationMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
Microsoft.AspNetCore.Builder.UseMiddlewareExtensions+<>c__DisplayClass6_1+<<UseMiddlewareInterface>b__1>d.MoveNext()
Umbraco.Cms.Web.BackOffice.Middleware.BackOfficeExternalLoginProviderErrorMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
Microsoft.AspNetCore.Builder.UseMiddlewareExtensions+<>c__DisplayClass6_1+<<UseMiddlewareInterface>b__1>d.MoveNext()
Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
Microsoft.AspNetCore.Session.SessionMiddleware.Invoke(HttpContext context)
Microsoft.AspNetCore.Localization.RequestLocalizationMiddleware.Invoke(HttpContext context)
Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
SixLabors.ImageSharp.Web.Middleware.ImageSharpMiddleware.Invoke(HttpContext context)
Microsoft.AspNetCore.Diagnostics.StatusCodePagesMiddleware.Invoke(HttpContext context)
StackExchange.Profiling.MiniProfilerMiddleware.Invoke(HttpContext context) in MiniProfilerMiddleware.cs
Umbraco.Cms.Web.Common.Middleware.UmbracoRequestMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
Microsoft.AspNetCore.Builder.UseMiddlewareExtensions+<>c__DisplayClass6_1+<<UseMiddlewareInterface>b__1>d.MoveNext()
Umbraco.Cms.Web.Common.Middleware.PreviewAuthenticationMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
Microsoft.AspNetCore.Builder.UseMiddlewareExtensions+<>c__DisplayClass6_1+<<UseMiddlewareInterface>b__1>d.MoveNext()
Umbraco.Cms.Web.Common.Middleware.UmbracoRequestLoggingMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
Microsoft.AspNetCore.Builder.UseMiddlewareExtensions+<>c__DisplayClass6_1+<<UseMiddlewareInterface>b__1>d.MoveNext()
Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
```

**Expected behavior**
CSS loads correctly.

**Environment (please complete the following information):**
 - Server OS: [e.g. iOS]
 - Umbraco Version [e.g. 9.0.0]
 - Product Version [e.g. 1.0.0]

**Additional context**
Fix appears to be changing the RGB definition of CSS as suggested in the linked article above from:

```
.konstrukt-card {
    display: flex;
    align-items: center;
    background-color: white;
    border-radius: 3px;
    padding: 15px;
    border: solid 1px #f5f5f5;
    **box-shadow: 0 1px 1px 0 rgb(0 0 0 / 16%);**
}
```

to:

```
.konstrukt-card {
    display: flex;
    align-items: center;
    background-color: white;
    border-radius: 3px;
    padding: 15px;
    border: solid 1px #f5f5f5;
    **box-shadow: 0px 1px 1px 0 rgba(0, 0, 0, 0.16);**
}
```

**Comment by mattbrailsford** [comment]
Hey @matt-tolliday-iss 

Good spot and thanks for the details on the workaround. I've updated this now and merged it into my 1.1.0 branch which will be released next week. You can try it out on our unstable feed if you want to ensure it works, but no worries if not. 

https://nuget.outfield.digital/unstable/v3/index.json

**Comment by mattbrailsford** [comment]
Doh! Sorry, I only looked at the change from `%`. I've pushed another update now so there should be another new build in a few minutes 👍
